keyboards/annepro2/keymaps/kdarkhan/README.md

```diff
@@ -0,0 +1,15 @@
+
+
+![picture of keymap](https://github.com/kdarkhan/qmk_firmware/raw/keyboard-annepro2-kdarkhan/keyboards/annepro2/keymaps/kdarkhan/personal-keymap.png)
```

Comment:
Hosting the image on my own fork because `png` files are ignored by `.gitignore`. Assuming that is to prevent size bloat of the repo. Not pushing the image here because of that.